vent/coordinator/rpc.py

```diff
@@ -0,0 +1,53 @@
+import xmlrpc.client
+import vent.controller.control_module
+from xmlrpc.server import SimpleXMLRPCServer
+import pickle
+
+default_addr = 'localhost'
+default_port = 9533
+
+remote_controller = None
+
+
+def get_sensors():
+    return pickle.dumps(remote_controller.get_sensors())
+
+
+def get_active_alarms():
+    return pickle.dumps(remote_controller.get_active_alarms())
+
+
+def get_logged_alarms():
+    return pickle.dumps(remote_controller.get_logged_alarms())
+
+
+def set_control(control_setting):
+    remote_controller.set_control(pickle.loads(control_setting.data))
+
+
+def get_control(control_setting_name):
+    return pickle.dumps(remote_controller.get_control(pickle.loads(control_setting_name.data)))
```

Comment:
Maybe don't nest calls so deep in one line, I get concerned it's harder to read and may be more error prone. I'm ok with the previous ones that only nest one deep although if we ever need to debug one of those calls we'll need to unnest it.

Reply:
I expended those.